templates/masterclasses.html

```diff
@@ -52,17 +51,12 @@ <h3 class="p-media-object__title">
                   <li class="p-media-object__meta-list-item--date">
                     <span class="u-off-screen">Date: </span>{{ session.Date.Formatted }}
                   </li>
-                  <!-- Add to calendar button, URL documentation: https://github.com/InteractionDesignFoundation/add-event-to-calendar-docs/blob/main/services/google.md -->
-                  <!-- Currently, all events start at 11AM CST with duration of 45 minutes -->
+                  <!-- Add to calendar btn uses a gcal published event https://support.google.com/calendar/answer/41207 -->
                   <li class="p-media-object__meta-list-item--date">
-                    <a href="https://calendar.google.com/calendar/render?action=TEMPLATE&text=Masterclass:+{{ session.Topic }}&dates={{ session.Date.Calendar }}T110000/{{ session.Date.Calendar }}T114500&ctz=Europe/Paris&location={{ session.Event }}&details={{ session.Owner }}+would+like+to+talk+about+{{ session.Topic }}.%0ANotes:+{{ session.Notes }}"
-                      target="_blank" rel="noopener noreferrer">
+                    <a href="{{ session.Event }}" target="_blank" rel="noopener noreferrer">
                       Add to Google Calendar
                     </a>
                   </li>
-                  <li class="p-media-object__meta-list-item--location">
-                    <span class="u-off-screen">Location: </span><a href="{{ session.Event }}">Google Meet</a>
```

Comment:
Why remove the meet link?

Reply:
Ah, I should’ve been more clear, I mentioned this in the PR description.
> Also removed the "Location" bullet point in the upcoming sessions column, because the published event already contains google meet link.